Sources/NIO/FileHandle.swift

```diff
@@ -84,11 +84,20 @@ extension NIOFileHandle {
     /// Open a new `NIOFileHandle`.
     ///
     /// - parameters:
-    ///     - path: the path of the file to open. The ownership of the file descriptor is transferred to this `NIOFileHandle` and so it will be closed once `close` is called.
-    public convenience init(path: String) throws {
+    ///     - forReadingAtPath: the path of the file to open for reading. The ownership of the file descriptor is transferred to this `NIOFileHandle` and so it will be closed once `close` is called.
+    public convenience init(forReadingAtPath path: String) throws {
```

Comment:
this is to mirror `Foundation's` `FileHandle`, do you think its a good idea?

Reply:
Well it makes this a semver major change, so at the very least you’ll have to return the pre-existing one. 

I don’t mind this change, but we need the old one. We can deprecate it though.

Reply:
Reverted reading path